grantnav/frontend/templates/help.html

```diff
@@ -240,6 +240,7 @@ <h3 id="words_phrases">{% trans "Words and phrases" %}</h3>
     <li>{% trans "Multiple words: e.g. <code>young people gardens</code> (each word does not necessarily have to be present)" %}</li>
     <li>{% trans "Require each word to be found: e.g. <code>young AND people</code>" %}</li>
     <li>{% trans 'An exact phrase by enclosing it in quotes: e.g. <code>"young people"</code>' %}</li>
+    <li>{% trans 'Exclude an organisation: e.g. <code>NOT fundingOrganization.name:"London Councils"</code>' %}</li>
```

Comment:
Maybe "Exclude results that contain a word: e.g. youth NOT clubs"

Reply:
@robredpath I have updated it.
What about `advanced_search.html`, is it being used?